src/Factories/JsonErrorFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Factories;
+
+use Throwable;
+use WtfPhp\JsonApiErrors\Models\JsonError;
+
+/**
+ * Class JsonErrorFactory
+ * @package WtfPhp\JsonApiErrors\Factories
+ */
+class JsonErrorFactory implements JsonErrorFactoryInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public static function createFromThrowable(Throwable $throwable, $status = 500): JsonError
+    {
+        $jsonError = new JsonError();
+        $jsonError->status = $status;
+        $jsonError->code = $throwable->getCode();
+        $jsonError->title = $throwable->getMessage();
+        $jsonError->detail = $throwable->getTraceAsString();
+        $jsonError->links = [
```

Comment:
The Docs say that links should only contain `about`: https://jsonapi.org/format/#error-objects